Client/Frontend/Toolbar+URLBar/URLBarView.swift

```diff
@@ -239,7 +239,7 @@ class URLBarView: UIView, URLBarViewProtocol, AlphaDimmable, TopBottomInterchang
             // Ecosia: Update URL accordingly
             // locationView.url = newURL
             var updatedUrl = newURL
-            if updatedUrl?.isEcosiaSearchQuery() == true {
+            if updatedUrl?.shouldEcosify() ?? false {
```

Comment:
Small suggestion when it comes to the readability point of view 👇 
```suggestion
            if updatedUrl?.shouldEcosify() == true {
```

Reply:
To be honest I prefer the other way since I believe it makes more clear what is the default value in case we have no url 😅

Reply:
Awesome then! I'm happy to leave it as is as don't have strong opinions 👍